src/scanner.coffee.md

```diff
@@ -139,14 +139,15 @@ of cells or null if the scanner is exhausted.
         return callback() if response and response.statusCode is 204
         return callback err if err
         cells = []
+        encoding = if @options.encoding is 'undefined' then @options.encoding else @client.options.encoding
```

Comment:
I think we shall simply add a default value for `@client.options.encoding`. In "src/client.coffee.md", it would look like: `options.encoding ?= 'utf8'` around line 55.